oc-chef-pedant/spec/api/policies/complete_endpoint_spec.rb

```diff
@@ -804,7 +804,7 @@ def rm(assoc)
             end
           end
 
-          context "when the name is close to the maximum size" do
+          context "when the name is close to the maximum size", :chef_zero_quirks do
 
             # On Chef Zero, some backends will append `.json` to a file name,
             # which can exceed the common limit of 255 characters.
```

Comment:
Probably worth cleaning up this comment and changing the line below back to 255 characters, since I only made it 250 in order to accommodate zero in the old implementation.